webrecorder/browsertrix
# View and edit browser profile

(https://github.com/webrecorder/browsertrix-cloud/issues/199) View and save edits to browser profile.

### Manual testing
1. Go to Browser Profiles and click on an item in the list. Verify you're taken to the detail page
2. Click "Launch Browser Profile". Verify dropdown shows list of origins
3. Choose origin and click "Start browser". Verify that browser starts up with chosen origin
4. Change name and description and click "Save Profile". Verify profile changes are saved.

### Screenshots
**Detail view:**
<img width="1023" alt="Screen Shot 2022-04-18 at 5 46 14 PM" src="https://user-images.githubusercontent.com/4672952/163898577-0cd84ede-2dcb-4d1d-ad2f-74a97eee6078.png">

**Launch dialog:**
<img width="514" alt="Screen Shot 2022-04-18 at 5 46 19 PM" src="https://user-images.githubusercontent.com/4672952/163898600-55ee44a9-3cac-4dfc-9b06-a387a946b700.png">

**Browser view:**
<img width="1018" alt="Screen Shot 2022-04-18 at 11 55 29 AM" src="https://user-images.githubusercontent.com/4672952/163860884-382ce120-c808-4761-877c-db94853ab3cb.png">
